### PR TITLE
Flashman 2: Your flash has NOT burned out!

### PR DESCRIPTION
**Pull request by ghost**
🆑 Cobby and a very tiny bit of armhulen
add: FLASH SUITS!! Hey, if you're a mascot of bright lights, this'll be right up and down your alley!
/🆑

Reasoning: It's a silly vanity item, and some people enjoy being able to dress up as wacky things. Unlike, say, the robbie rotten meme outfit, dressing up as a parody version of a weapon in the game will always be in good taste and not just a monthly meme.

I NEED TO MOVE OVER THE SPRITE ITSELF AND I'M ADDING IT TO MAINT. PATIENCE PEONS.

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -1. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Hatterhat** [comment]
fuck off GBP machine, @Armhulen is a good coder; better than @kevinz000 who just makes shitcode

**Comment by ExcessiveUseOfCobblestone** [comment]
hahahaa

**Comment by Armhulen** [comment]
The only reason i'm in the negative is because my fix prs are tweaks and my features have no bugs in them to fix

**Comment by Militaires** [comment]
#24407 follow this execution and sprites

as well as the name and desc

**Comment by Armhulen** [comment]
oh okay sure

**Comment by Armhulen** [comment]
oh that looks so swell

**Comment by duncathan** [comment]
i'm bullying you

**Comment by ExcessiveUseOfCobblestone** [comment]
DELETE THAT CLOSE

**Comment by Hatterhat** [comment]
fuck you @duncathan

**Comment by duncathan** [comment]
the numbers don't lie

**Comment by Armhulen** [comment]
@duncathan please refer to my lawyer @optimumtact he has my offshore token account

**Comment by Armhulen** [comment]
i hear if you say oranges into a pr 3 times he shows up, like a discount bloody mary

**Comment by duncathan** [comment]
i'll give you a loan of two tokens and reopen if you open a fix pr

**Comment by Armhulen** [comment]
deal

**Comment by Armhulen** [comment]
@duncathan fix pr is up!

**Comment by duncathan** [comment]
good work

**Comment by Anonmare** [comment]
**F E A R
T H E
F L A S H**

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -572,3 +572,20 @@
 	icon_state = "bedsheet"
 	user_vars_to_edit = list("name" = "Spooky Ghost", "real_name" = "Spooky Ghost" , "incorporeal_move" = INCORPOREAL_MOVE_BASIC, "appearance_flags" = KEEP_TOGETHER|TILE_BOUND, "alpha" = 150)
 	alternate_worn_layer = ABOVE_BODY_FRONT_LAYER //so the bedsheet goes over everything but fire
+
+/obj/item/clothing/suit/hooded/flashsuit
+	name = "flash suit"
+	desc = "a suit that parodies those who use the defense item of the same name."
```
Comment (RandomMarine) [inline code/modules/clothing/suits/miscellaneous.dm]: This description could be much better.

**Comment by Kraseo** [comment]
imagen
no, but a tin can

**Comment by Armhulen** [comment]
@Kraseo :)

**Comment by kevinz000** [comment]
@Hatterhat and you don't code LMAO

**Comment by Armhulen** [comment]
i need to save my tokens at the moment

**Comment by ExcessiveUseOfCobblestone** [comment]
lol

**Pull request closed without merging**